.github/workflows/doc-preview.yaml

```diff
@@ -0,0 +1,56 @@
+on:
+  issue_comment:
+    types: [created]
+
+name: doc-preview.yaml
+
+permissions: read-all
+
+jobs:
+  preview:
+    if: ${{ github.event.issue.pull_request && (github.event.comment.author_association == 'COLLABORATOR' || github.event.comment.author_association == 'CONTRIBUTOR' || github.event.comment.author_association == 'MEMBER' || github.event.comment.author_association == 'OWNER') && startsWith(github.event.comment.body, '/preview-docs') }}
+
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
```

Comment:
question: what are these permissions for?  [This example](https://github.com/nwtgck/actions-netlify?tab=readme-ov-file#usage) doesn't seem to need to add permissions at all, and neither do any of our existing workflows somehow?  (Maybe something connected with [this](https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token)?  But I thought at some point we did need to specify permissions:...)  And I would have expected some middle ground, like `pull-requests: write` to POST a comment and `contents: read` for the checkout.

Reply:
That example has `enable-pull-request-comment: false` and we don't, so yea you're right we need `pull-requests: write` write the comment and we probably only need `contents: read`, let me test in the other repo.